src/server/services/afis/afis-facturen.ts

```diff
@@ -174,13 +216,15 @@ function transformFactuur(
   const factuurDocumentIdEncrypted = factuurDocumentId
     ? encryptSessionIdWithRouteIdParam(sessionID, factuurDocumentId)
     : null;
-
-  const netPaymentAmountInCents = parseFloat(invoice.NetPaymentAmount) * 100;
+  const MULTIPLY_WITH_100 = 100;
+  const netPaymentAmountInCents =
+    parseFloat(invoice.NetPaymentAmount) * MULTIPLY_WITH_100;
   const amountInBalanceTransacCrcyInCents =
-    parseFloat(invoice.AmountInBalanceTransacCrcy) * 100;
+    parseFloat(invoice.AmountInBalanceTransacCrcy) * MULTIPLY_WITH_100;
 
   const amountOwed =
-    (amountInBalanceTransacCrcyInCents + netPaymentAmountInCents) / 100;
+    (amountInBalanceTransacCrcyInCents + netPaymentAmountInCents) /
```

Comment:
is dan weer een beetje raar bij een deling, maar wat moet je anders, ONE_IN_CENTS?